### DISCUSSION TITLE
System wide configuration so that users are not prompted on first run

### DISCUSSION THREAD

**Discussion by thenewguy** [d2phap/ImageGlass#1412] [Q&A] [answered]
I installed ImageGlass for the first time to see how it works and both user accounts I tested were prompted for configuration selections on startup.  I would like to make the selections for all users and avoid this prompt.

How does one configure ImageGlass for all users at once?

**Comment by d2phap** [comment, marked as answer]
You can create `igconfig.admin.xml` file and put it in the installed folder where ImageGlass.exe is located, with this content:

```xml
<ImageGlass>
  <Configuration>
    <Info description="ImageGlass Configuration file" version="7.5" />
    <Content>
      <Item key="FirstLaunchVersion" value="999999" />
    </Content>
  </Configuration>
</ImageGlass>
```

This file is to lock the settings from the users. For more information, please visit: https://imageglass.org/docs/app-configs

---------

### Edit: Visit https://github.com/d2phap/ImageGlass/issues/1680#issuecomment-1775177046 if you're using ImageGlass version 9

**Reply by CastyMcBoozer** [reply]
This had no effect for me.

**Reply by d2phap** [reply]
The above is for v8 which uses XML format. V9 uses JSON format, pls refer to https://github.com/d2phap/ImageGlass/issues/1680#issuecomment-1775177046

**Reply by CastyMcBoozer** [reply]
Thanks.
